Nest/00. Plan.md

```diff
@@ -0,0 +1,58 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Synchronizing the local cache
+### 2.2.2 Common issues
+
+## 2.3 Installing and uninstalling a package
+### 2.3.1 Installing a package
+### 2.3.2 Uninstalling a package
+### 2.3.3 Common issues
+
+## 2.4 Updating the OS
+### 2.4.1 Common issues
+
+## 2.5 Summary
+
+# 3 Advanced concepts
+## 3.1 Managing repositories
+### 3.1.1 The official repositories
+### 3.1.2 The unofficial repositories
+### 3.1.3 Configuring repositories
+
+## 3.2 Overview of the cache
+
+## 3.3 Version requirement
+
+# 4 Finest
+## 4.1 Key concepts
+### 4.1.1 The cache in detail
+### 4.1.2 The transactions
+### 4.1.3 The dependency graph
+### 4.1.4 Groups
+### 4.1.5 Example of transactions with the dependency graph
+
+## 4.2 Finest's options
+
+## 4.3 Finest's commands
+### 4.3.1 Managing packages through requirements
+
+### 4.3.2 Updating the OS by merging the dependency graph
+
+### 4.3.3 Group management
```

Comment:
I'd rather put "Managing groups" here, since the other items in this section all use verbs

Reply:
Me too :P (https://github.com/raven-os/getting-started/pull/1#discussion_r279592496)